src/main/kotlin/br/unipar/plano/domain/reembolso/model/ReembolsoRejeitado.kt

```diff
@@ -0,0 +1,23 @@
+package br.unipar.plano.domain.reembolso.model
+
+import java.time.LocalDate
+import java.util.*
+import javax.persistence.*
+
+@Entity
+class ReembolsoRejeitado (
```

Comment:
Melhor descrever como substantivo do que como verbo.

```suggestion
class RejeicaoReembolso (
```